### PR TITLE
select-chart-type

### PR DESCRIPTION
**Pull request by YavorLalev**
The user can select the type of chart.

[Link to the US-2]( https://github.com/users/catdieval/projects/1/views/1?pane=issue&itemId=54594681)

### CODE REVIEW

File: utils/scatter.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+      <text
+        x="0"
+        y="120"
+        fill="#000000"
+        font-size="5px"
+        font-weight="bold"
+        font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif"
+      >
+        from the Noun Project
+      </text> */}
```
Comment (MerleMak) [inline utils/scatter.js:207]:
```suggestion
```

File: utils/lineplotplusmarkers.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <text
+        x="0"
+        y="148"
+        fill="#000000"
+        font-size="5px"
+        font-weight="bold"
+        font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif"
+      >
+        from the Noun Project
+      </text> */}
```
Comment (MerleMak) [inline utils/lineplotplusmarkers.js:38]:
```suggestion
```

File: utils/lineplot.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      <text
+        x="0"
+        y="62"
+        fill="#000000"
+        font-size="5px"
+        font-weight="bold"
+        font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif"
+      >
+        from the Noun Project
+      </text> */}
```
Comment (MerleMak) [inline utils/lineplot.js:34]:
```suggestion
```

File: utils/barplot.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      <text
+        x="0"
+        y="120"
+        fill="#000000"
+        font-size="5px"
+        font-weight="bold"
+        font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif"
+      >
+        from the Noun Project
+      </text> */}
```
Comment (MerleMak) [inline utils/barplot.js:36]:
```suggestion
```

File: Components/FlexContainer/index.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+export const FlexContainer = styled.section`
+  flex-direction: ${(props) =>
+    props.$direction === "column" ? "column" : "row"};
```
Comment (MerleMak) [inline Components/FlexContainer/index.js:5]: nice!!

File: Components/ChartItem/index.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  );
+
+  function handleClick(event) {
+    setClickedChartType(event.target.innerText);
+    alert(`You chose ${name}`);
+  }
+
+  return (
+    <FlexContainer $direction="column">
+      <Button onClick={handleClick}>{name}</Button>
```
Comment (MerleMak) [inline Components/ChartItem/index.js:19]:
```suggestion
      <Button onClick={handleChartType}>{name}</Button>
```

File: Components/ChartItem/index.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+import Image from "next/image";
+import { FlexContainer } from "../FlexContainer";
+
+export default function ChartItem({ name, icon }) {
+  const [clickedChartType, setClickedChartType] = useLocalStorageState(
+    "clickedChartType",
+    { defaultValue: "" }
+  );
+
+  function handleClick(event) {
```
Comment (MerleMak) [inline Components/ChartItem/index.js:12]:
```suggestion
  function handleChartType(event) {
```

**Review by MerleMak — COMMENTED** [review]
really Nice job!

I think you are not using the SVGs in the utils and the chart-types-svg folders, right?
If you really don't use them you can delete them to not clutter your project :)

Keep uop the goof work, this is a nice PR!

File: Components/ChartItem/index.js (hunk 7 above)
Comment (YavorLalev) [reply Components/ChartItem/index.js]: Done.

File: Components/ChartItem/index.js (hunk 6 above)
Comment (YavorLalev) [reply Components/ChartItem/index.js:19]: Done.

File: utils/barplot.js (hunk 4 above)
Comment (YavorLalev) [reply utils/barplot.js:36]: Done.

File: utils/scatter.js (hunk 1 above)
Comment (YavorLalev) [reply utils/scatter.js:207]: Done.

File: utils/lineplot.js (hunk 3 above)
Comment (YavorLalev) [reply utils/lineplot.js:34]: Done.

File: Components/FlexContainer/index.js (hunk 5 above)
Comment (YavorLalev) [reply Components/FlexContainer/index.js:5]: 😀

**Pull request merged**